events_lab.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let form = document.querySelector("form");
+    let list = document.createElement("ul");
+    let errorMsg = document.createElement("p");
+    
+    
+    form.addEventListener("submit", (e) => {
+        let userInput = document.querySelector("#str").value;
+        e.preventDefault();
+        if(userInput === ""){
```

Comment:
using the method `.trim()` will give you a more reliable outcome, since `"   "` would not be the same as `""`and im sure we dont want to add that either to the list